features/channel/managing_channels/adding_channel_with_a_toggle_for_the_lowest_price.feature

```diff
@@ -0,0 +1,52 @@
+@managing_channels
+Feature: Choosing whether to show the lowest product price or not while creating a channel
+    In order to show the lowest price before the product has been discounted only for certain channels
+    As an Administrator
+    I want to add a new channel that has this feature enabled or not
+
+    Background:
+        Given the store has currency "Euro"
+        And the store has locale "English (United States)"
+        And the store operates in "United States" and "Poland"
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Adding a new channel with lowest price from 30 days before the product has been discounted enabled by default
```

Comment:
I was hoping that you would remove `30 days` everywhere in your scenarios 😅

Reply:
Hah, I didn't know I have it in scenarios also – will fix that later :+1: